### PR TITLE
[ACS-5576]  Icon updated as per the figma

### PR DESCRIPTION
**Pull request by pkunduGL**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Currently  left_chevron and close icon class were  used in the toolbar component and info drawer.

**What is the new behaviour?**
Now the icon classes  are updated as per the figma.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://alfresco.atlassian.net/browse/ACS-5576

### CODE REVIEW

File: projects/aca-shared/src/lib/components/toolbar/toolbar.component.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <mat-toolbar>
   <ng-container *ngFor="let item of items; trackBy: trackByActionId">
-    <aca-toolbar-action [actionRef]="item"></aca-toolbar-action>
+    <aca-toolbar-action [ngClass]="{'rotate-icon': item.icon === 'unfold_more'}" [actionRef]="item"></aca-toolbar-action>
```
Comment (DenysVuika) [inline projects/aca-shared/src/lib/components/toolbar/toolbar.component.html:3]: this is incorrect, you are hacking all actions to have static icon

Comment (DenysVuika) [reply projects/aca-shared/src/lib/components/toolbar/toolbar.component.html:3]: if there's more than one icon here, are you going to hardcode dozens of icon names?

Comment (DenysVuika) [reply projects/aca-shared/src/lib/components/toolbar/toolbar.component.html:3]: solution: just register the svg image with the icon pointing to the right direcion

File: projects/aca-shared/src/lib/components/toolbar/toolbar.component.scss [hunk 2]
```diff
@@ -25,3 +25,7 @@ aca-toolbar {
     margin-right: 5px;
   }
 }
+
+.rotate-icon {
```
Comment (DenysVuika) [inline projects/aca-shared/src/lib/components/toolbar/toolbar.component.scss:29]: this is childish... just register the proper image pointing to the expected direction

File: projects/aca-shared/src/lib/components/toolbar/toolbar.component.html (hunk 1 above)
Comment (pkunduGL) [reply projects/aca-shared/src/lib/components/toolbar/toolbar.component.html]: Implemnted the suggested solutiuon.

File: projects/aca-shared/src/lib/components/toolbar/toolbar.component.scss (hunk 2 above)
Comment (pkunduGL) [reply projects/aca-shared/src/lib/components/toolbar/toolbar.component.scss]: Registered the svg

**Review by arohilaGL — APPROVED** [review]

**Review by nikita-web-ua — APPROVED** [review]

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**